public/help/skins-basics.html

```diff
@@ -1,19 +1,14 @@
 <p>
-  A site skin lets you change the way the Archive looks in your browser, so you can customize your reading experience.
+  A site skin lets you change the way the Archive is presented when you are logged in to your account.
```

Comment:
I can change it to "when you are logged in," but the "is presented" change is because skins aren't just visual, but aural and tactical, too.